package.json

```diff
@@ -22,29 +22,29 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "aws-sdk": "^2.603.0",
-    "aws-xray-sdk": "^2.5.0",
-    "js-yaml": "^3.13.1",
+    "aws-sdk": "^2.745.0",
```

Comment:
Since dependencies issues have been identified with npm audit, I will create a ticket to bump the packages version

Reply:
https://jira.dvsacloud.uk/browse/CVSB-18222
ticket created @MartiUK , thanks for the upgrade